app/controllers/stripe-setup/stripe-setup.util.js

```diff
@@ -72,25 +71,12 @@ async function getExistingResponsiblePersonName (account, isSwitchingCredentials
 }
 
 async function completeKyc (gatewayAccountId, service, stripeAccountId, correlationId) {
-  let telephoneNumber = lodash.get(service, 'merchantDetails.telephone_number')
-  let formattedPhoneNumber
-
-  if (telephoneNumber) {
-    formattedPhoneNumber = formatPhoneNumberWithCountryCode(telephoneNumber)
-  }
-
-  const [stripeAccountResponse] = await Promise.all([
-    addNewCapabilities(stripeAccountId, service.merchantDetails.name, formattedPhoneNumber),
+  await Promise.all([
     connector.setStripeAccountSetupFlag(gatewayAccountId, 'additional_kyc_data', correlationId),
     connector.disableCollectAdditionalKyc(gatewayAccountId, correlationId)
   ])
 
-  const stripeLegacyPayments = lodash.get(stripeAccountResponse, 'capabilities.legacy_payments')
-  if (stripeLegacyPayments === 'active') {
-    await removeLegacyPaymentsCapability(stripeAccountId)
-  }
-
-  logger.info('KYC additional information completed for Stripe account', {
+  logger.info('KYC additional information completed for Stripe account (but not moved to new capabilities)', {
```

Comment:
Do we want to continue adding the additional information - MCC, product description, telephone so that when we or Stripe come to change it, it's only the capabilities that need updating?

Reply:
good point! updated. Also included few changes from here https://github.com/alphagov/pay-selfservice/pull/3127